desktop-widgets/mainwindow.cpp

```diff
@@ -1131,34 +1129,33 @@ void MainWindow::on_actionViewList_triggered()
 {
 	toggleCollapsible(true);
 	beginChangeState(LIST_MAXIMIZED);
-	ui.mainSplitter->setSizes(BEHAVIOR << COLLAPSED << EXPANDED);
-	ui.bottomSplitter->setSizes(BEHAVIOR << EXPANDED << COLLAPSED);
+	ui.mainSplitter->setSizes({COLLAPSED, EXPANDED});
```

Comment:
can we please, use spaces after `{` and before `}` for the lists - e.g.:
`{ COLLAPSED, EXPANDED }`

Reply:
done.